core/src/main/java/org/opensearch/sql/data/model/ExprDateValue.java

```diff
@@ -25,6 +31,7 @@
 public class ExprDateValue extends AbstractExprValue {
 
   private final LocalDate date;
+  private String dateFormat;
```

Comment:
add javadoc, what is dateFormat? is it a Format? or formatted date?

Reply:
Renamed this to formattedDate and added java doc.